treeherder/model/migrations/0012_add_related_names.py

```diff
@@ -0,0 +1,82 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.3 on 2017-07-21 20:25
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('model', '0011_create_group'),
+    ]
+
+    operations = [
+        migrations.RunSQL(
+            migrations.RunSQL.noop,
+            reverse_sql=migrations.RunSQL.noop,
+            state_operations=[
+                migrations.AlterField(
```

Comment:
Was this because Django created a buggy migration? What was the output of sqlmigrate on it?
If so, could you clarify via a comment here and file a bug upstream about it?